doc/policies/authentication.rst

```diff
@@ -587,3 +587,17 @@ supported by the token.
 
 .. note:: If you configure this, you will likely also want to configure
     :ref:`policy_webauthn_enroll_user_verification_requirement`.
+
+
+question_numbers
```

Comment:
Let's call this "question_number" or better "num_questions" since it is only one number and many qustions.

Reply:
Will do "question_number", since the first part is the name of the tokentype "question".

Reply:
Done in 2ed7dea9